### PR TITLE
Cfae spring cleaning

### PR DESCRIPTION
**Pull request by Casperfrc**

### CODE REVIEW

File: src/database/sql/audio.py [hunk 1]
```diff
@@ -210,9 +231,10 @@ def get_all(self, audio_id: str) -> str:
         """
 
         query = """
-                SELECT * 
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                SELECT *
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 2]
```diff
@@ -233,8 +255,9 @@ def get_rhythm(self, audio_id: str) -> str:
 
         query = """
                 SELECT bpm, bpm_confidence, last_updated
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 3]
```diff
@@ -255,8 +278,9 @@ def get_bpm(self, audio_id: str) -> str:
...
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 4]
```diff
@@ -278,7 +302,8 @@ def get_timbre(self, audio_id: str) -> str:
         query = """
                 SELECT timbre, timbre_confidence, last_updated
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 5]
```diff
@@ -298,11 +323,14 @@ def get_emotions(self, audio_id: str) -> str:
...
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                SELECT relaxed, relaxed_confidence, party,
+                       party_confidence, aggressive,
+                       aggressive_confidence, happy,
+                       happy_confidence, sad,
+                       sad_confidence, last_updated
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 6]
```diff
@@ -324,7 +352,8 @@ def get_relaxed(self, audio_id: str) -> str:
         query = """
                 SELECT relaxed, relaxed_confidence, last_updated
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 7]
```diff
@@ -346,7 +375,8 @@ def get_party(self, audio_id: str) -> str:
         query = """
             SELECT party, party_confidence, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 8]
```diff
@@ -368,7 +398,8 @@ def get_aggressive(self, audio_id: str) -> str:
         query = """
             SELECT aggressive, aggressive_confidence, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 9]
```diff
@@ -390,7 +421,8 @@ def get_happy(self, audio_id: str) -> str:
         query = """
             SELECT happy, happy_confidence, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 10]
```diff
@@ -412,7 +444,8 @@ def get_sad(self, audio_id: str) -> str:
         query = """
                 SELECT sad, sad_confidence, last_updated
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 11]
```diff
@@ -434,7 +467,8 @@ def get_level(self, audio_id: str) -> str:
         query = """
             SELECT peak, Loudness_integrated, loudness_range, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 12]
```diff
@@ -456,7 +490,8 @@ def get_peak(self, audio_id: str) -> str:
         query = """
             SELECT peak, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 13]
```diff
@@ -478,7 +513,8 @@ def get_loudness_integrated(self, audio_id: str) -> str:
         query = """
             SELECT Loudness_integrated, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/database/sql/audio.py [hunk 14]
```diff
@@ -500,7 +536,8 @@ def get_loudness_range(self, audio_id: str) -> str:
         query = """
             SELECT loudness_range, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```
Comment (Areedtz) [inline src/database/sql/audio.py]: If you want this AND `audio_track=:track` to be on a new line, please indent it.

File: src/rest_api/general_application.py [hunk 15]
```diff
@@ -632,3 +634,11 @@ def check_if_none(result, diskotek_nr):
...
         )
     return
+
+
+def format_date(result):
+    date = datetime.datetime.strptime(
+            result['Last_Updated'], '%Y-%m-%dT%H:%M:%S')
+    result['Last_Updated'] = date.isoformat()
+    
+    return result
```
Comment (Areedtz) [inline src/rest_api/general_application.py]: I don't think this function is used. Please remove it.

File: src/rest_api/music_application.py [hunk 16]
```diff
@@ -11,6 +8,11 @@
 
 from tasks import check_done, add_bpm, add_emotions, add_metering, add_similarity_features, save_to_db
 import classification.api_helper as music_emotion_classifier
+# Some linting can show this as not being imported
+from celery import chain
+
+from tasks import check_done, add_bpm, add_emotions, add_metering, \
+    add_similarity_features, save_to_db
 from utilities.config_loader import load_config
 from utilities.get_song_id import get_song_id
```
Comment (Areedtz) [inline src/rest_api/music_application.py]: Look at the imports.

I don't this comment is needed:
`# Some linting can show this as not being imported`

File: src/rest_api/music_application.py [hunk 17]
```diff
@@ -69,7 +71,8 @@ def post(self) -> str:
         if os.path.isfile(song_path):
             add_to_pipeline(data, song_path)
         else:  # Is folder
-            for file in os.listdir(song_path):
+            for _ in os.listdir(song_path):
                 add_to_pipeline(data, song_path)
```
Comment (Areedtz) [inline src/rest_api/music_application.py]: Something wrong here.

File: src/similarity/t/test_split_song.py [hunk 18]
```diff
@@ -1,13 +1,12 @@
+# Some linting might say these imports fail
```
Comment (Areedtz) [inline src/similarity/t/test_split_song.py]: This comment should be removed.

File: src/utilities/filehandler/audio_loader.py [hunk 19]
```diff
@@ -1,3 +1,4 @@
+# Some linting might say these imports fail
```
Comment (Areedtz) [inline src/utilities/filehandler/audio_loader.py]: This comment should be removed.

File: src/video_emotion/facial_recognition/t/test_facial_recognition.py [hunk 20]
```diff
@@ -1,13 +1,13 @@
 from video_emotion.facial_recognition.facial_recognition import analyze_video
 from utilities.filehandler.handle_path import get_absolute_path
 
+
 def test_facial_recognition_of_at_least_one_face():
-    test_filename = get_absolute_path("video_emotion/facial_recognition/t/te"
-                                      + "st_facial_recognition/Fun_at_a_Fair.mp4")
+    test_filename = (get_absolute_path
+                     ("video_emotion/facial_recognition/t/te" +
+                      "st_facial_recognition/Fun_at_a_Fair.mp4"))
```
Comment (Areedtz) [inline src/video_emotion/facial_recognition/t/test_facial_recognition.py]: Would probably (if possible) make the string cut differently so the "test" word isn't cut up.

File: src/database/mongo/audio/song_segment.py [hunk 21]
```diff
@@ -157,7 +162,11 @@ def get_all_in_range(self, from_count: int, to_count: int):
         """
 
         results = []
-        for r in self._db._db[self._dbcol].find().limit(to_count - from_count).skip(from_count):
+        for r in self._db._db[
+                              self._dbcol
+                              ].find().limit(
+                                             to_count - from_count
+                                             ).skip(from_count):
```
Comment (Areedtz) [inline src/database/mongo/audio/song_segment.py]: I think this looks really weird. Don't know how to do it better though

File: src/database/sql/audio.py [hunk 22]
```diff
@@ -1,6 +1,8 @@
 from typing import Dict
 import json
 
+# Some linting might say this doesn't get imported
+# It does.
```
[the remark this answers is not in the document]
Comment (Casperfrc) [reply src/database/sql/audio.py]: Removed

File: src/database/sql/audio.py [hunk 23]
```diff
@@ -76,13 +79,14 @@ def exists(self, audio_id: str) -> bool:
         query = """
                 SELECT audio_release
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```
[the remark this answers is not in the document]
Comment (Casperfrc) [reply src/database/sql/audio.py]: Removed

File: src/database/sql/audio.py (hunk 1 above)
Comment (Casperfrc) [reply src/database/sql/audio.py]: Removed

File: src/database/sql/audio.py (hunk 2 above)
Comment (Casperfrc) [reply src/database/sql/audio.py]: Removed

File: src/database/sql/audio.py (hunk 3 above)
Comment (Casperfrc) [reply src/database/sql/audio.py]: Removed

File: src/rest_api/music_application.py (hunk 17 above)
Comment (Casperfrc) [reply src/rest_api/music_application.py]: Fixed

File: src/similarity/t/test_split_song.py (hunk 18 above)
Comment (Casperfrc) [reply src/similarity/t/test_split_song.py]: Done

File: src/utilities/filehandler/audio_loader.py (hunk 19 above)
Comment (Casperfrc) [reply src/utilities/filehandler/audio_loader.py]: Done

File: src/video_emotion/facial_recognition/t/test_facial_recognition.py (hunk 20 above)
Comment (Casperfrc) [reply src/video_emotion/facial_recognition/t/test_facial_recognition.py]: Done

File: src/database/mongo/audio/song_segment.py (hunk 21 above)
Comment (Casperfrc) [reply src/database/mongo/audio/song_segment.py]: Found a different way that my PEP8 linter eats.

**Pull request merged**